frontend/src/components/table/index.ts

```diff
@@ -0,0 +1,6 @@
+export * from './types';
+export * from './Table';
```

Comment:
I think this should be:
```suggestion
export { default as Table } from './Table'; 
```

Personally I don't think we need to remove the default exports for each file.